mailpoet/.env.sample

```diff
@@ -44,3 +44,19 @@ WP_TEST_PERFORMANCE_DATA_URL=
 WP_TEST_PERFORMANCE_PW=
 K6_CLOUD_TOKEN=
 K6_CLOUD_ID=
+
+# Begin git hooks configuration
+MP_GIT_HOOKS_ENABLE=true
+
+MP_GIT_HOOKS_ESLINT=true
+
+MP_GIT_HOOKS_STYLELINT=true
+
+MP_GIT_HOOKS_PHPLINT=true
+MP_GIT_HOOKS_CODE_SNIFFER=true
+MP_GIT_HOOKS_MINIMAL_PLUGIN_STANDARDS=true
+MP_GIT_HOOKS_PHPSTAN=true
+
+MP_GIT_HOOKS_INSTALL_JS=true
+MP_GIT_HOOKS_INSTALL_PHP=true
+# End git hooks configuration
```

Comment:
This is just a tiny stylistic note, but the begin/end comments seem to be rather inconsistent with the rest of the file. I think a single block with a single heading above is fine:

```sh
# Git hooks configuration
MP_GIT_HOOKS_ENABLE=true
MP_GIT_HOOKS_ESLINT=true
MP_GIT_HOOKS_STYLELINT=true
MP_GIT_HOOKS_PHPLINT=true
MP_GIT_HOOKS_CODE_SNIFFER=true
MP_GIT_HOOKS_MINIMAL_PLUGIN_STANDARDS=true
MP_GIT_HOOKS_PHPSTAN=true
MP_GIT_HOOKS_INSTALL_JS=true
MP_GIT_HOOKS_INSTALL_PHP=true
```

Reply:
I agree, updated in 244c9ab73f839fa955f9288679926d851b9c0d64